figma-import.config.js.example

```diff
@@ -0,0 +1,12 @@
+/**
+ * FIGMA_TOKEN - your personal API token, generated in figma profile settings
+ * FILE_KEY - unique file id
+ * FRAME_WITH_ICONS_ID - frame id with assets to export
+ * e.g. https://www.figma.com/file/[FILE_KEY]/Joystream-Atlas?node-id=[FRAME_WITH_ICONS_ID]
+ */
+
+module.exports = {
+  FIGMA_TOKEN: '',
+  FILE_KEY: '',
+  FRAME_WITH_ICONS_ID: '',
+}
```

Comment:
I wanted to test it out but I still didn't know what to provide for the last two fields of config. I think we should either provide detailed description on what to select in Figma or just provide those values supplied here already

Reply:
I don't know how to write this without sounding passive aggressive 😛, but TBH I'm not sure what is unclear in the description above, and I think providing values here is not a good idea because, we might commit `figma-import.config.js` as well. Everything you need is in the figma url with icons, as I showed in the example here https://github.com/Joystream/atlas/blob/835367d89084df82fd095c82e7bb76c2608709a3/figma-import.config.js.example#L5
Anyway I improved description a little bit

Reply:
So I see the following problems:
1. It's not immediately clear which file should be used to extract icons.
2. I have no idea what `FRAME_WITH_ICONS_ID` should be, i.e. how I get this value. We should make sure that someone that isn't able to ask this questions is able to figure it out by themselves so I think we need to improve the instructions a bit.

> I think providing values here is not a good idea because, we might commit figma-import.config.js as well

I don't really understand what you mean by that.

Reply:
Please see if it's better now

Reply:
Looks good, thanks for improving this